tests/helpers.py

```diff
@@ -17,22 +37,19 @@ def gen_random_page(num_elements=None, start_element=0, count=None,
         "start_element": start_element,
         "num_elements": num_elements,
         "count": count,
-        object_type: [gen_random_object() for _ in range(num_elements)]
+        object_type: [object_generator_func(start_element + index)
+                      for index in range(num_elements)]
     }
 
 
-def gen_random_collection(count=None, object_type="campaigns"):
-    if count is None:
-        count = random.randrange(10000)
-    result = []
-    i = 0
-    for i in range(count // 100):
-        random_page = gen_random_page(count=count, object_type=object_type,
-                                      start_element=i * 100)
-        result.append(random_page)
-    if count % 100 != 0:
-        random_page = gen_random_page(count=count, object_type=object_type,
-                                      start_element=i * 100,
-                                      num_elements=count % 100)
-        result.append(random_page)
-    return result
+def gen_random_collection(start_element=0, count=None,
+                          object_type="campaigns"):
+    return gen_collection(
+        object_generator_func=lambda index: {"id": random.randrange(1000000)},
+        start_element=start_element, count=count, object_type=object_type)
+
+
+def gen_ordered_collection(start_element, count, object_type="campaigns"):
+    return gen_collection(
+        object_generator_func=lambda index: {"id": index},
+        start_element=start_element, count=count, object_type=object_type)
```

Comment:
I guess it would be more readable/maintainable to add a `random: bool` parameter to `gen_collection` and `gen_page` than having these two very similar functions, what do you think?

Reply:
Agreed, changes made